examples/example-deno-runtime/tests.ts

```diff
@@ -295,6 +294,11 @@ const imports: Imports = {
       },
     });
   },
+
+  importStructWithOptions: (arg: StructWithOptions): StructWithOptions => {
+    assertEquals(arg.potentiallyOptionalString, "Hello");
```

Comment:
I think we should also add a test case for the omitted value, so we verify whether it's `null` or `undefined`.